.idea/workspace.xml

```diff
@@ -4,7 +4,11 @@
     <option name="autoReloadType" value="ALL" />
   </component>
   <component name="ChangeListManager">
-    <list default="true" id="8356264b-efb4-4c68-883b-f6b1a1f23dfb" name="Changes" comment="" />
```

Comment:
Does it file should be ignored? or is it useful to share?

Reply:
I'll add this to git ignore!